model.js

```diff
@@ -191,6 +191,24 @@ export class Model {
     return this._findByCriteria(criteria, this.edges);
   }
 
+  strands() {
+    const smallStrandSize = 315;
+    const bigStrandSize = 360
```

Comment:
Ideally, we don't assume any specific numbers in this file since we want it to be generic and infer things from the model.json.

Reply:
Yeah I thought about that too. What if I passed these numbers in from patterns.toml like so:
`options = { small_strand_size = "315", big_strand_size="360"}`

and then had these be arguments to this function. That way by default it uses the magic numbers for our current dragon egg, but the numbers can be changed through configs